rest/persistent_config_test.go

```diff
@@ -1236,3 +1236,103 @@ func makeDbConfig(bucketName string, dbName string, scopesConfig ScopesConfig) D
 	}
 	return dbConfig
 }
+
+func TestPersistentConfigNoBucketField(t *testing.T) {
+	base.TestsRequireBootstrapConnection(t)
+	base.RequireNumTestBuckets(t, 2)
+
+	base.SetUpTestLogging(t, base.LevelTrace, base.KeyConfig)
```

Comment:
remove trace logging here? No strong feelings about this, probably won't log a lot.